packages/plugin-web-attribution-browser/src/web-attribution.ts

```diff
@@ -26,7 +26,13 @@ export const webAttributionPlugin: CreateWebAttributionPlugin = function (option
         storage.get(storageKey),
       ]);
 
-      if (isNewCampaign(currentCampaign, previousCampaign, pluginConfig)) {
+      const currentTime = Date.now();
+      const lastEventTime = config.lastEventTime || Date.now();
+      const timeSinceLastEvent = currentTime - lastEventTime;
+
+      const isNewSession = timeSinceLastEvent > config.sessionTimeout;
```

Comment:
let's create a shared function that checks for new session between browser SDK and this plugin to make sure they always yield the same results

Reply:
where is the good place to declare the shared function? In analytics-client-common?

Reply:
yup i think so

Reply:
moved this in analytics-client-common.